### PR TITLE
Re-organiza CWL version outputs and fixes:

### PR DESCRIPTION
**Pull request by mberacochea**
- Rename outputs from PPR-Meta, vifinder and virsorter
- HMMER db Direcoty issue, refactor to use individual files instead
- Code formatting for some scripts

### CODE REVIEW

File: cwl/src/Tools/FastaRename/fasta_restore.cwl [hunk 1]
```diff
@@ -23,7 +23,9 @@ baseCommand: ["python", "rename_fasta.py"]
 
 inputs:
   input:
-    type: File?
+    type:
```
[the remark this answers is not in the document]
Comment (caballero) [reply cwl/src/Tools/FastaRename/fasta_restore.cwl:26]: 'File?' means the 'input' can be not defined in the yaml, and use the value from the expression, it is ugly but it is CWL valid

**Review by caballero — APPROVED** [review]

File: cwl/src/Tools/FastaRename/fasta_restore.cwl (hunk 1 above)
[the remark this answers is not in the document]
Comment (mberacochea) [reply cwl/src/Tools/FastaRename/fasta_restore.cwl:26]: Is either a File (so it has to exists) or a File? (it's optional). The types are CWL types.

[the remark this answers is not in the document]
Comment (KeteSakharova) [reply cwl/src/Tools/FastaRename/fasta_restore.cwl:26]: Why you can't leave only File? instead of [File, File?]?

**Review by KeteSakharova — APPROVED** [review]

File: cwl/src/Tools/FastaRename/fasta_restore.cwl (hunk 1 above)
[the remark this answers is not in the document]
Comment (mberacochea) [reply cwl/src/Tools/FastaRename/fasta_restore.cwl:26]: I get a warning, that bit is used by a step that sends File and a different step sends File?.
File and File? are different "types" for CWL

**Pull request merged**